javascript-light-bulb/main.js

```diff
@@ -0,0 +1,15 @@
+var brightnessClick = 0;
+var $circle = document.querySelector('.circle');
+var $body = document.querySelector('body');
+function circleClick(event) {
+  brightnessClick++;
+  if (brightnessClick % 2 === 0) {
```

Comment:
Instead of using a counter variable and just checking if the current count is even or odd, there is a better data type that has only two states that you can use to determine if the light is "on" or not: a boolean.